notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>Also here, after the initial example and this explanation, I would show that the conditional statement gives a boolean series, which is then used to filter the dataframe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I agree with @jorisvandenbossche. This is very confusing for beginners, and so it would be helpful to show the mechanics as he described.</p>